### ISSUE TITLE
Is tensorboardX good to use with Keras

### ISSUE THREAD

**Issue by nobodykid** [lanpa/tensorboardX#510]
Hello,

This might be a very simple and obvious question, but I haven't seen anyone asking for it. So I want to clarify if I could use this with Keras. 
The Keras' version of Tensorboard only works through callbacks and it's more intuitive using tensorboardX since I'm running my loss and gradient calculation manually (so not using `fit()`, which had args for `callbacks` where you put the Tensorboard callback)

**Comment by lanpa** [comment]
I am afraid that it might be incompatible especially if you are using the tensorflow backend. Can you print the value you want to log without too much effort under Keras? If the answer is yes, I think it might not too hard to log the data with tensorboardX.

**Comment by nobodykid** [comment]
I see. I intended to use the histogram visualizer and projector for analyzing my embedding matrix. 

Used to work with Pytorch before, so I plan to use the same method on Keras. But apparently Keras' Tensorboard API only available through callbacks, and I run a specific loss calculation rather than using `model.fit()`

**Comment by lanpa** [comment]
If you use tensorflow as Keras backend, I think both the histogram the projector should be supported natively. Good luck!
